src/de/jost_net/JVerein/gui/menu/EigenschaftMenu.java

```diff
@@ -33,8 +33,8 @@ public class EigenschaftMenu extends ContextMenu
    */
   public EigenschaftMenu()
   {
-    addItem(new ContextMenuItem("Neu", new EigenschaftDetailAction(true),
-        "document-new.png"));
+    addItem(new CheckedSingleContextMenuItem("Bearbeiten", new EigenschaftDetailAction(false),
+        "text-x-generic.png"));
     addItem(new CheckedContextMenuItem("L�schen",
```

Comment:
Bei Eigernschaft Löschen erscheint kein Abfragedialog ob man wirklich löschen will. Konsistenter Weise sollte man das auch hier machen.

Reply:
Das habe ich in #333 gemacht